dms-infrastructure/src/main/kotlin/team/aliens/dms/global/security/SecurityConfig.kt

```diff
@@ -175,6 +175,8 @@ class SecurityConfig(
                 .requestMatchers(HttpMethod.PATCH, "/outings/{outing-application-id}").hasAuthority(MANAGER.name)
                 .requestMatchers(HttpMethod.GET, "/outings/files").hasAuthority(MANAGER.name)
                 .requestMatchers(HttpMethod.GET, "/outings/my").hasAuthority(STUDENT.name)
+                .requestMatchers(HttpMethod.GET, "/outings").hasAuthority(MANAGER.name)

```

Comment:
```suggestion
                .requestMatchers(HttpMethod.GET, "/outings/histories").hasAuthority(MANAGER.name)

```
histories도 같이 사용하면 더 알아보기 쉬울 것 같아요